control_scripting

```diff
@@ -0,0 +1,16 @@
+Package: ni-scan-engine-ethercat-veristand-{veristand_version}-labview-support
```

Comment:
The custom device package name is `ni-scan-engine-veristand-{veristand_version}-support`, so this should omit `ethercat-`.

```suggestion
Package: ni-scan-engine-veristand-{veristand_version}-labview-support
```